esp/esp/themes/theme_data/fruitsalad/templates/main.html

```diff
@@ -175,6 +179,9 @@
   <a href="/myesp/home/">more announcements...</a></div>
 {% endif %}
 {% else %}
+<!-- Default blank if there's no announcements -->
+{% endif %}
+{% else %}
 <h1 class="title">{% block content_title %}&nbsp;{% endblock %}</h1>
```

Comment:
If you move this line above ``{% if request.path|equal:"/teach/index.html" or request.path|equal:"/learn/index.html" %}`` on line 144, then teach/index.html and learn/index.html will display their QSD page titles. Then, when the announcements area is hidden, the page will look exactly the same as all other QSD pages.